common/src/models/request/identityToken/apiTokenRequest.ts

```diff
@@ -13,10 +13,10 @@ export class ApiTokenRequest extends TokenRequest {
   }
 
   toIdentityToken(clientId: string) {
-    const obj = super.toIdentityToken(clientId);
+    // Use the api clientId provided by the user in the TokenRequest instead of the method argument
```

Comment:
😂 That's fair enough. The real problem here - which made the comment necessary - is that we were taking an argument that we were then ignoring, just to fit the same method signature as the other `tokenRequest` classes. I've now fixed this in [this commit](https://github.com/bitwarden/jslib/pull/588/commits/8e8cb3241badbbf7bc9b60af06dc9121e0aa1ade) so that we just have a different method signature in this subclass.